lib/mix/tasks/phx.gen.auth.ex

```diff
@@ -115,6 +115,7 @@ defmodule Mix.Tasks.Phx.Gen.Auth do
       schema: schema,
       migration: migration,
       hashing_library: hashing_library,
+      web_app_name: Phoenix.Naming.underscore(context.web_module),
```

Comment:
This causes a dialyzer error because `Phoenix.Naming.underscore/1` expects a string instead of an atom. How about making this a private function like `router_scope/1`or `web_app_prefix/1`?

```elixir
defp web_app_name(%Context{} = context) do
  context.web_module
  |> to_string()
  |> Phoenix.Naming.underscore()
end
```